fixtures/samples/infected/009.php

```diff
@@ -0,0 +1,2 @@
+<?php
+$ca2a = @$GLOBALS[$GLOBALS['k94124e4'][48].$GLOBALS['k94124e4'][
```

Comment:
In this case, this is not a "clean code" case, but a not working at all code. :-)